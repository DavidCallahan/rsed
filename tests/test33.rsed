foreach all  
    print $CURRENT
    print length($CURRENT)
    print length($CURRENT) < 3
end
