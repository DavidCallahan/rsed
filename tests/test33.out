1
1
true
2
1
true
3 more
6
false
4
1
true
5 less
6
false
6
1
true
